cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -896,12 +896,12 @@ public void testLoadBalancerExternalTrafficPolicyLocalFromTemplate() {
 
         // Check external bootstrap service
         Service ext = kc.generateExternalBootstrapServices().get(0);
-        assertThat(ext.getSpec().getExternalTrafficPolicy(), is(ExternalTrafficPolicy.LOCAL.toValue()));
+        assertThat(ext.getSpec().getExternalTrafficPolicy(), is(nullValue()));
```

Comment:
I guess you can just remove these test since it was there just test this feature. Just make sure we have the tests for the current was how to set it.